manifest.json

```diff
@@ -17,11 +17,5 @@
   ],
   "background": {
     "scripts": ["background.js"]
-  },
-  "licenses": [
```

Comment:
Removal of this section will break building and possibly unlicense some of the output.